### ISSUE TITLE
Add another `comments` string on Crowdin

### ISSUE THREAD

**Issue by dakeshi** [busyorg/busy#1446]
Currently, there are only one comments on Crowdin. But I need to provide different translation depends on the its location. There are two cases that comments string is used:

One is a title for writing comments on the posting.
![_fabien_-_discord](https://user-images.githubusercontent.com/8563047/35661925-a3e0d2c8-0758-11e8-932d-c17f9915ba17.png)

Another comments strings is different meaning. This is related to comments I wrote.
![_fabien_-_discord](https://user-images.githubusercontent.com/8563047/35661993-ffd61214-0758-11e8-84b6-df136647fe06.png)

And it will be helpful you add context with a screenshot for those of strings. (refer to #1445 )

**Comment by Sekhmet** [comment]
I don't think they are different.

They both refer to list of comments (first one is not a title for writing comments, but for list of comments).

Does it make sense to have the same string in this case?

**Comment by dakeshi** [comment]
No. They are different context. The later one is the list of comments I wrote. The former one is for listing of comments for the posting.

I keep try to match with korean steemit community translation so if it is translated same words, some user can be confused about it.
